src/views/workflow-summary-tab/workflow-summary-tab-json-view/workflow-summary-tab-json-view.types.ts

```diff
@@ -0,0 +1,4 @@
+export type Props = {
+  inputJson: any;
```

Comment:
nit: since this is JSON, can we pass it in as an object type?

Reply:
Json can be array, string, boolean etc
But i can change it for a better type that accepts only serializable Objects